images/airflow/2.10.0/python/mwaa/subprocess/subprocess.py

```diff
@@ -188,7 +188,7 @@ def start(
                 exc_info=sys.exc_info(),
             )
 
-    @throttle(seconds=60, instance_level_throttling=True) # avoid excessive calls to process conditions
```

Comment:
It looks like a few of the recent changes (https://github.com/aws/amazon-mwaa-docker-images/pull/136, https://github.com/aws/amazon-mwaa-docker-images/pull/135) haven't been included here. Probably requires a rebase.

Reply:
Fixed this